### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://translate.mattermost.com/projects/mattermost/mattermost-mobile_master/) for mattermost-languages-shipped/mattermost-mobile_master.

### CODE REVIEW

File: assets/base/i18n/fr.json [hunk 1]
```diff
@@ -552,8 +552,8 @@
   "mobile.sidebar_settings.permanent": "Barre latérale permanente",
   "mobile.share_extension.too_long_title": "Le message est trop long",
   "mobile.share_extension.too_long_message": "Nombre de caractères : {count}/{max}",
-  "mobile.server_ssl.error.title": "Untrusted Certificate",
-  "mobile.server_ssl.error.text": "The certificate from {host} is not trusted.\n\nPlease contact your System Administrator to resolve the certificate issues and allow connections to this server.",
+  "mobile.server_ssl.error.title": "",
```
Comment (enahum) [inline assets/base/i18n/fr.json:555]: @wget any reason why these are left empty?

File: assets/base/i18n/ko.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
   "about.teamEditiont0": "팀 에디션",
   "about.teamEditiont1": "엔터프라이즈 에디션",
   "about.title": "{appTitle} 정보",
-  "announcment_banner.dont_show_again": "Don't show again",
+  "announcment_banner.dont_show_again": "",
```
Comment (enahum) [inline assets/base/i18n/ko.json:13]: @metanerd can this be a problem with Weblate? some french strings are empty and korean has a bunch of them empty

File: assets/base/i18n/uk.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
 {
   "about.date": "Дата створення:",
-  "about.enterpriseEditione1": "Enterprise Edition",
+  "about.enterpriseEditione1": "",
```
Comment (enahum) [inline assets/base/i18n/uk.json:3]: same with Ukrainian now

**Comment by enahum** [comment]
If some strings have been removed from the webapp, then the same work should be done on the mobile app... who is responsible for that ?

**Comment by enahum** [comment]
@wiersgallak @esethna would you know ?

File: assets/base/i18n/fr.json (hunk 1 above)
Comment (metanerd) [reply assets/base/i18n/fr.json:555]: That was me, all good. Will close this PR to revert.

**Comment by wiersgallak** [comment]
@esethna Can you please determine what would be the best process for ensuring the mobile app is also updated?

**Comment by esethna** [comment]
@enahum sorry I don't have context on this change and am not familiar with the translation process. @metanerd would you know?

**Pull request closed without merging**